apps/opik-frontend/src/components/pages/ExperimentsPage/charts/ExperimentChartContainer.tsx

```diff
@@ -0,0 +1,157 @@
+};
+
+const ExperimentChartContainer: React.FC<ExperimentChartContainerProps> = ({
+  chartData,
+  className,
+}) => {
+  const [hiddenLines, setHiddenLines] = useState<string[]>([]);
+
+  const config = useMemo(() => {
+    return chartData.lines.reduce<ChartConfig>((acc, line) => {
+      acc[line] = {
+        label: line,
+        color: TAG_VARIANTS_COLOR_MAP[generateTagVariant(line)!],
+      };
+      return acc;
+    }, {});
+  }, [chartData.lines]);
+
+  const tickWidth = useMemo(() => {
+    const MIN_WIDTH = 26;
+    const MAX_WIDTH = 80;
+    const CHARACTER_WIDTH = 7;
+    const EXTRA_SPACE = 10;
+
+    const values = chartData.data.reduce<number[]>((acc, data) => {
+      return [
+        ...acc,
+        ...Object.values(data.scores).map(
+          (v) => Math.round(v).toString().length,
+        ),
+      ];
+    }, []);
+
+    return Math.min(
+      Math.max(MIN_WIDTH, Math.max(...values) * CHARACTER_WIDTH + EXTRA_SPACE),
+      MAX_WIDTH,
+    );
+  }, [chartData.data]);
+
+  const [width, setWidth] = useState<number>(0);
+  const { ref } = useObserveResizeNode<HTMLDivElement>((node) =>
+    setWidth(node.clientWidth),
+  );
+
+  const {
+    dataset: { id: chartId, name },
+  } = chartData;
+
+  const legendWidth = Math.max(
+    MIN_LEGEND_WIDTH,
+    Math.min(width * 0.3, MAX_LEGEND_WIDTH),
+  );
+
+  return (
+    <Card className={cn("min-w-[max(400px,40%)]", className)} ref={ref}>
+      <CardHeader>
+        <CardTitle>{name}</CardTitle>
+      </CardHeader>
+      <CardContent>
+        <ChartContainer config={config} className="h-32 w-full">
+          <LineChart
+            data={chartData.data}
+            margin={{ top: 10, bottom: 10, left: 0, right: 0 }}
+          >
+            <CartesianGrid vertical={false} />
+            <YAxis
+              width={tickWidth}
+              axisLine={false}
+              tickLine={false}
+              tick={{
+                stroke: "#94A3B8",
+                fontSize: 10,
+                fontWeight: 200,
+              }}
+              interval="preserveStartEnd"
+            />
+            <ChartTooltip
+              cursor={false}
+              isAnimationActive={false}
+              content={<ExperimentChartTooltipContent />}
+            />
+            <ChartLegend
+              verticalAlign="top"
+              layout="vertical"
+              align="right"
+              content={
+                <ExperimentChartLegendContent
+                  setHideState={setHiddenLines}
+                  chartId={chartId}
+                />
+              }
+              width={legendWidth}
+              height={128}
+            />
+            {chartData.lines.map((line) => {
+              const hide = hiddenLines.includes(line);
+
+              return (
+                <Line
+                  type="natural"
+                  key={line}
+                  isAnimationActive={false}
+                  dataKey={(x) => x.scores[line] || undefined}
```

Comment:
Why we don't display the `0` data points? Not sure what we're trying to achieve with this `|| undefined` here

Reply:
It was some testing code that I forgot to delete, thanks for poining me.